### DISCUSSION TITLE
for retrieving database image file

### DISCUSSION THREAD

**Discussion by jaipalsin** [bcit-ci/CodeIgniter#6297] [Q&A]
### hello everyone i,m using php,s codeigniter framework I want retrieve database file image but always return null value please fix this issue
`
public function getOrder($id)
	{
$this->db->select('i.*, p.image, p.productname, p.price');
		$this->db->from($this->ordItemsTable . ' as i');
		$this->db->join($this->proTable . ' as p', 'p.id = i.product_id', 'left');
		$this->db->where('i.order_id', $id);
		$query2 = $this->db->get();
		$result['items'] = ($query2->num_rows() > 0) ? $query2->result_array() : array();

		// Return fetched data
		return !empty($result) ? $result : false;
} ?>
`

**Comment by bunglegrind** [comment]
There's no issue on CI3. Maybe you have an issue on your code. Or on your data.

**Reply by jaipalsin** [reply]
but what the issue on this code can you guide me

**Reply by bunglegrind** [reply]
Try without the query builder. Write the sql using the query method. Is it working now?
